collect/central_collector.go

```diff
@@ -630,6 +656,10 @@ func (c *CentralCollector) send(status *centralstore.CentralTraceStatus) {
 			sp.Data["meta.refinery.send_reason"] = TraceSendLateSpan
 
 		}
+		sp.Data["meta.span_event_count"] = int(status.SpanEventCount())
```

Comment:
Did we decide to add counts to every span? And make it not optional? We'll have to change this in config too.

Reply:
I believe so unless my memory is messing with me which is not a uncommon thing